app/api/resources/mentorship_relation.py

```diff
@@ -24,6 +26,8 @@
 
 @mentorship_relation_ns.route("mentorship_relation/send_request")
 class SendRequest(Resource):
+    decorators = [limiter.limit(rate_limits.LIMIT_1)]
```

Comment:
@jalajcodes , I don't see this variable "decorators" being used anywhere in the code. Do we need to declare it as variable? Can we use the `pie syntax` (`@`) instead just like we used the @email_verification_required in MS. You can define the limiter decorator function inside the same file (the app/utils/decorator_utils.py).